msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/Authority.java

```diff
@@ -76,6 +80,9 @@ static AuthorityType detectAuthorityType(URL authorityUrl) {
 
         final String path = authorityUrl.getPath().substring(1);
         if (StringHelper.isBlank(path)) {
+            if(isCiamAuthority(authorityUrl.getHost())){
```

Comment:
Why does `path` have to blank here? If `authorityurl` is `https://msidlabciam1.ciamlogin.com/d57fb3d4-4b5a-4144-9328-9c1f7d58179d/` then the `path` is `d57...` and this is not blank?

Reply:
To close the conversation here, if the path is not blank the CIAM authority type will be detected at Line 97 of the class.